cityofaustin/atd-airflow
# Add dag for internal School Zone Beacons dataset

I piggybacked a handful of typo fixing in the PR

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/13946

## Associated repo

https://github.com/cityofaustin/atd-knack-services/pull/122/

## Testing

**Steps to test:**

here is the dataset, it is an internal one, so you will have to log in to see it
https://datahub.austintexas.gov/Transportation-and-Mobility/School-Zone-Beacons-Internal/8whb-sg4d

---
#### Ship list
